### DISCUSSION TITLE
create the nzp for all devices :)/crear el nzp para todas las consolas:)

### DISCUSSION THREAD

**Discussion by Ibrahimprox1** [nzp-team/nzportable#1033] [General]
Estoy pensando en ser parte del team del nzp, y tengo la idea de encargarme de hacer el nzp para otras consolas y móviles, por ejemplo PS2 o PS3, Xbox 360, móviles y tablets,
Debido a esto quiero ser parte del team de nzp para encargarme de los otros dispositivos y tema de optimización

In English:
I'm thinking about being part of the nzp team, and I have the idea of being in charge of making the nzp for other consoles and mobile phones, for example PS2 or PS3, Xbox 360, mobile phones and tablets, Because of this I want to be part of the nzp team to take care of the other devices and optimization issues.

Atentamente, Ibrahim
sincerely, ibrahim

**Comment by MotoLegacy** [comment]
Please do not create duplicate discussions. This project is open source, you do not need to be on any sort of team to be able to port this project wherever you'd like.
